packages/cli/src/utils.ts

```diff
@@ -146,6 +146,15 @@ export const  getConfigWithChecks = async (
 
   validateLocalizableResources(config);
 
+  const googlePkg = config.solana_mobile_dapp_publisher_portal.google_store_package;
+  if (googlePkg != "" && googlePkg.length > 0) {
```

Comment:
Rather than `!= ""`, should this be checking for a truth-y value here? Something like:
```
if (googlePkg && googlePkg.length > 0) {
```

That would exclude missing fields (which should be allowed) and empty fields.

Reply:
or even more succinct, `if (googlePkg?.length) {`